assets/js/hocs/with-searched-products.js

```diff
@@ -20,7 +20,7 @@ const withSearchedProducts = createHigherOrderComponent( ( OriginalComponent ) =
 	 * A Component wrapping the passed in component.
 	 *
 	 * @class WrappedComponent
-	 * @extends {Component}
+	 * @augments {Component}
```

Comment:
huh, I'm surprised by this one.  I've not seen the use of `@augments` until now (unless I just never noticed before).

Reply:
You're right.

Default configs were complaining about `extends`, but it's ok now that we are using WordPress defaults, so I undid it in a74781c1c828382f403d9b723253a8c5c0631897.